packages/perseus/src/widgets/__tests__/python-program.ts

```diff
@@ -0,0 +1,43 @@
+import "@testing-library/jest-dom";
+
+import {testDependencies} from "../../../../../testing/test-dependencies";
+import * as Dependencies from "../../dependencies";
+import {question1} from "../__testdata__/python-program.testdata";
+
+import {renderQuestion} from "./renderQuestion";
+
+describe("python-program widget", () => {
+    beforeEach(() => {
+        jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
+            testDependencies,
+        );
+    });
+
+    it("should snapshot", () => {
+        // Arrange
+        const apiOptions = {
+            isMobile: false,
+        } as const;
+
+        // Act
+        const {container} = renderQuestion(question1, apiOptions);
+
+        // Assert
+        expect(container).toMatchSnapshot("first render");
+    });
+
+    it("should snapshot on mobile", () => {
+        // Arrange
+        const apiOptions = {
+            isMobile: true,
+        } as const;
+
+        // Act
+        const {container} = renderQuestion(question1, apiOptions);
+
+        // Assert
+        expect(container).toMatchSnapshot("first mobile render");
+    });
+
+    // This widget doesn't have any direct behavior, it just renders an iframe
```

Comment:
Thanks for leaving this here. One challenge we haven't tackled well in Perseus yet is dealing with Stories that use frames.